config/__init__.py

```diff
@@ -118,7 +118,7 @@ def __init__(self, missing_config_fields):
         """
         Initialize a new MissingConfigError.
 
-        :param: missing_config_fields List of missing config variables
+        :param missing_config_fields: the missing config variables

```

Comment:
is there a linter that can catch these documentation errors?

Reply:
I doubt it.